src/crafting_gui.cpp

```diff
@@ -202,9 +198,35 @@ const recipe *select_crafting_recipe( int &batch_size )
             if( batch ) {
                 batch_recipes( crafting_inv, helpers, current, available, chosen );
             } else {
-                // Set current to all recipes in the current tab; available are possible to make
-                pick_recipes( crafting_inv, helpers, current, available,
-                              tab.cur(), subtab.cur(), filterstring );
+
+                available.clear();
+                current.clear();
```

Comment:
Dupe of [these lines](https://github.com/CleverRaven/Cataclysm-DDA/pull/18286/files/967ce23bf96e41b7013fb2f48ddbbab19a0f95cc#diff-fc40e9036f0ff584805625e294aadbb0R196).